tt_metal/hostdevcommon/common_runtime_address_map.h

```diff
@@ -119,6 +127,9 @@ static constexpr uint32_t CQ_CONSUMER_CB1_TOTAL_SIZE = CQ_CONSUMER_CB1_PAGE_SIZE
 static constexpr uint32_t CQ_CONSUMER_CB1_READ_PTR = CQ_CONSUMER_CB1_TOTAL_SIZE + L1_ALIGNMENT;
 static constexpr uint32_t CQ_CONSUMER_CB1_WRITE_PTR = CQ_CONSUMER_CB1_READ_PTR + L1_ALIGNMENT;
 
+constexpr static std::uint32_t L1_UNRESERVED_BASE = CQ_CONSUMER_CB_BASE + CQ_CONSUMER_CB_ADDR_RESERVATION; //CQ_CONSUMER_CB_BASE + 16 * L1_ALIGNMENT; // Start of unreserved space
+constexpr static std::uint32_t ERISC_L1_UNRESERVED_BASE = L1_UNRESERVED_BASE; // Start of unreserved space
```

Comment:
Is `ERISC_L1_UNRESERVED_BASE` used anywhere?

Reply:
yes its used for idle erisc dispatch feature.
Currently its being set to same value as tensix dispatcher but may change in future.